gard_owl_ingest/analysis/mondo_mapping_status.py

```diff
@@ -1,7 +1,11 @@
-"""Mapping status between GARD and Mondo"""
+"""Mapping status between GARD and Mondo
```

Comment:
@matentzn Feel free to examine the code in this latest analysis if you like.

@hrsdhgd @souzadevinicius You guys as well, feel free. The gist here is that we're trying to align Mondo with GARD. This is some analytical code where we are taking a CSV that GARD just supplied us, and we're comparing it with existing Mondo mappings, and also trying to ascertain proxy mappings through Orphanet.